tests/smb-eicar-file-nbss-more-ffsmb/README.md

```diff
@@ -0,0 +1,15 @@
+# Description
+
+Test SMB EICAR file rule.
+
+# PCAP
+
+The pcap comes from running Linux client smbclient against a Windows 2019 Server (with a shared folder public without needed authentication)
+
+Needs a Proxy that sends two netbios requests in one segment
```

Comment:
Do you mean two NetBIOS messages in one TCP packet ?

Reply:
Yes, that's it. I edited it

Reply:
Still waiting for the `git push` or new PR